### ISSUE TITLE
Getting started code doesn't work with stable version (1.7.5)

### ISSUE THREAD

**Issue by synotic** [dc-js/dc.js#1060]
I'm not sure if this is expected, but took me some time to debug this. This is what the table from the homepage looks like on 1.7.5:

<img width="536" alt="screen shot 2015-12-08 at 3 34 50 pm" src="https://cloud.githubusercontent.com/assets/42359/11671997/44fce97a-9dc1-11e5-8467-fa5c0938ee4d.png">

and on the latest version (as expected):

<img width="259" alt="screen shot 2015-12-08 at 3 35 49 pm" src="https://cloud.githubusercontent.com/assets/42359/11672011/609d8162-9dc1-11e5-9eb3-283977978037.png">

bl.ocks 1.7.5: https://bl.ocks.org/synotic/3531c03b28552050a701
bl.ocks latest: http://bl.ocks.org/synotic/2860744098f3b19e1b2c

The only difference between the examples should be the version. I'm not sure if this expected or not, but I expected the Getting Started guide on the main page to work with the stable version.

**Comment by gordonwoodhull** [comment]
Hi @synotic, thanks for the report. 

Indeed a feature was added to the dataTable API has changed and the example was updated - I guess you could say the root problem is that the web site is running the 2.0 beta and not the stable version. It has been kind of an endless beta since there were a number of regressions due to hastily-checked-in code which we're still trying to recover from. But it is now API stable and I encourage you to try it out.

Or, you can find the annotated source for the 1.7.* stock example here:
https://github.com/dc-js/dc.js/blob/1.7/web/stock.js

Once we finally release 2.0 and it becomes the stable version, we will be sure to keep the web site pointing there instead of moving forward with dev. Thanks again for pointing this out.

**Comment by synotic** [comment]
Got it, thanks. I'm using 2.0 and it's working well. I just created this ticket because in the meantime it's probably pretty confusing to beginners :).

**Comment by gordonwoodhull** [comment]
Yes, thanks @synotic, hopefully this issue helps searchers. We'll get on track to a normal stable / develop setup soon.
